sys-process/systemd-cron/systemd-cron-1.5.12-r1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
New ebuilds should use EAPI 6 unless there's something that currently breaks with it.